### DISCUSSION TITLE
CSS Color Module Level 4 support

### DISCUSSION THREAD

**Discussion by RXminuS** [tailwindlabs/tailwindcss#4306] [Ideas]
Related discussion: https://github.com/tailwindlabs/tailwindcss/discussions/2218

Basically, adding the ability to add `display-p3` colour variants to the generated CSS.

```
.pink-text {
  color: rgb(255, 0, 79); /* Will be used as a fallback */
  color: color(display-p3 1 0 0.331); /* Will be used if supported */
}
```

I tried doing this as a plugin but got a bit lost in the weeds. In particular, I couldn't figure out the way to hook into the generation of CSS (in particular for colours).  Happy to implement this myself with some pointers but this might be relatively easy to add with some PostCSS plugins?

**Comment by RXminuS** [comment]
Derp! So sorry, I completely misunderstood Discussions / Issues work. I thought the original discussion had gone unanswered because it was not made into an issue. But now I've just double posted.

This one can and probably should be closed in favor of #2218. Sorry again

**Reply by barrycenter** [reply]
I just responded with a detailed (but flawed) workaround in #2218.
